site/src/store/slices/transferCreditsSlice.ts

```diff
@@ -1,17 +1,11 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
-import { APExam, TransferredGE } from '@peterportal/types';
+import { APExam, TransferredGE, UserAPExam } from '@peterportal/types';
 
 export interface TransferredCourse {
   courseName: string;
   units: number;
 }
```

Comment:
I'd argue that these other types should also be refactored, and that the 3 should be named consistently

`TransferredCourse`, `TransferredAPExam`, `TransferredGE`, `TransferredUncategorizedItem` (idk last one is kinda weird)

and they should _all_ be in `@peterportal/types` for consistency. Even better, they should be consolidated with the types currently defined in the transfer credits controller by using `z.infer`